cypher-shell/src/integration-test/java/org/neo4j/shell/commands/CypherShellPlainIntegrationTest.java

```diff
@@ -67,6 +69,24 @@ public void cypherWithProfileStatements() throws CommandException {
         assertThat(actual, containsString("NULL"));
     }
 
+    @Test
+    public void cypherWithProfileWithMemory() throws CommandException {
+        // given
+
+        String serverVersion = shell.getServerVersion();
+        // Memory profile are only available from 4.1
+        assumeTrue( majorVersion( serverVersion ) >= 4);
```

Comment:
Not important but I guess you could do

```java
assumeThat( version(serverVersion), greaterThanOrEqualTo(version("4.1")));
```

`Version` implements `Comparable<Version>` .

Reply:
Well, actually this would fail for 5.0, so I guess I should change it.

I can't find `greaterThanOrEqualTo` in CoreMatchers though. I guess I need some other hamcrest dependency.

Reply:
Oh ok, or you could just use `assumeTrue(version(serverVersion).compareTo(version("4.1")) >= 0)`